src/auto-evo/RunResults.cs

```diff
@@ -703,6 +847,13 @@ public class SpeciesResult
         {
             public Species Species;
 
+            /// <summary>
+            ///   Dictionary of the new species population for relevant patches,
+            ///   limited to natural increase/decrease and emergence.
+            /// </summary>
+            /// <remarks>
+            ///   Does not consider migrations.
```

Comment:
Needs to be in a para, also this doesn't consider split populations, which are similar to migrations that you need to tell the code to dynamically calculate those when needed